components/__tests__/CopyButton-test.js

```diff
@@ -60,12 +60,12 @@ describe('CopyButton', () => {
   describe('Renders feedback as expected', () => {
     it('Should make the feedback visible', () => {
       const feedbackWrapper = mount(<CopyButton feedback="Copied!" />);
-      const feedback = feedbackWrapper.find('.bx--btn--copy__feedback');
-      expect(feedback.hasClass('bx--btn--copy__feedback--displayed')).toBe(
+      const feedback = () => feedbackWrapper.find('.bx--btn--copy__feedback');
```

Comment:
Given that this pattern is coming up across files, it could potentially be even more DRY:

```js
const getBySelector = (sel, wrapper) => wrapper.find(sel);

// Or, curried:
const getBySelector = sel => wrapper => wrapper.find(sel);
```

The curried version would let you do:
```js
const getInput = getBySelector('input');
expect(getInput(wrapper))...
```

Might be overkill, but could also be a handy way to consolidate this kind of lookup logic in case the API changes again in enzyme@4.